src/core/user/user.spec.js

```diff
@@ -866,6 +866,17 @@ describe('User', () => {
       const user = new User();
       return chai.expect(user.me()).to.eventually.be.rejected;
     });
+
+    it.only('should fail if there is no active user', () => {
+      return UserMock.logout()
+        .then(() => User.me())
+        .then(() => {
+          throw new Error('This should not fail');
```

Comment:
I know it's not really part of the test, but I think this error message should be more accurate. We do expect this test to throw an exception, please change the string to something like `This test should not successfully resolve, it should throw an error.`